src/Interpreters/Context.h

```diff
@@ -2,6 +2,7 @@
 
 #include <base/types.h>
 #include <Core/Block_fwd.h>
+#include <Common/ThreadStatus.h>
```

Comment:
Why? I don't see any other changes being done in the file so adding new includes seems odd. Note that the less includes in `Context.h` the better